### PR TITLE
Added verify email template

### PR DESCRIPTION
**Pull request by ModernOctave**
Added verify email template as per #1

### CODE REVIEW

File: CDC_Backend/templates/verify_email.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                </tr>
+                <tr>
+                    <td style="padding:36px 30px 42px 30px;">
+                        <table role="presentation"
+                               style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+                            <tr>
+                                <td style="padding:0 0 36px 0;color:#153643;">
+                                    <h1 style="font-size:24px;margin:0 0 20px 0;font-family: 'Roboto', sans-serif;">Hi,</h1>
+                                    <p style="margin:0 0 12px 0;font-size:16px;line-height:24px;font-family: 'Roboto', sans-serif;">
+                                        You have successfully filled the job notification form for <b>{{ job_designation }}</b> at <b>{{ company_name }}</b>. Kindly verify your email by clicking <a href={{ one_time_link }}>here</a>.
```
Comment (gowtham3105) [inline CDC_Backend/templates/verify_email.html:46]: Change the Message to this
`We have received your Job Notification for {{designation}}. Kindly verify your email by clicking <a href={{ one_time_link }}>here</a>`

**Review by gowtham3105 — CHANGES_REQUESTED** [review]
Input Fields are
`designation` and `one_time_link`

File: CDC_Backend/templates/verify_email.html (hunk 1 above)
Comment (ModernOctave) [reply CDC_Backend/templates/verify_email.html]: Resolved in commit a9b128d

**Review by gowtham3105 — APPROVED** [review]
Approved

**Pull request merged**